src/TemplatesBuilder.js

```diff
@@ -127,6 +127,10 @@ export default class TemplatesBuilder {
     const propertyTemplate = this.newBasePropertyTemplate(propertyTerm)
     propertyTemplate.type = "literal"
     propertyTemplate.defaults = this.defaultsForLiteral(propertyTerm)
+    propertyTemplate.validationRegex =
```

Comment:
This needs a test. (I know you're not done, but thought I would point out just in case.)

Reply:
will do.